### PR TITLE
New sample exercising memmap on Zephyr (Arduino 101)

### PR DESCRIPTION
**Pull request by glima**
Signed-off-by: Gustavo Lima Chaves gustavo.lima.chaves@intel.com

### CODE REVIEW

File: soletta-memmap/config.zephyr/sol.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+NETWORK=n
+USE_IPM=n
+#FLASH_MEM_REGION_OFFSET=0xff000
```
Comment (edersondisouza) [inline soletta-memmap/config.zephyr/sol.conf:3]: Why this commented setting?

**Comment by edersondisouza** [comment]
Maybe a simple explanation of the sample on readme would be good?

File: soletta-memmap/config.zephyr/sol.conf (hunk 1 above)
Comment (glima) [inline soletta-memmap/config.zephyr/sol.conf:3]: I'll remove

**Comment by glima** [comment]
D'accord, will do.

**Comment by glima** [comment]
Fixed and merged.

**Pull request closed without merging**